docs/SRS/SRS.tex

```diff
@@ -752,24 +757,195 @@ \section{Project Issues}
+
+There are many libraries and frameworks available that can be reused to accelerate the building
+process of the application. Next.js can be used to provide a framework to build the application.
+Next.js comes with many out-of-box solutions for common website development problems. NextAuth.js
+is a library designed to help simplify the authentication process. Prisma is a library designed to
+help simplify the communication between the application and the database. Next-PWA is a library
+designed to quickly bootstrap a Next.js application into a progressive web application.
+
+A runtime and ecosystem that is available to use is Node.js. This runtime comes with a large
+ecosystem of packages that can be reused and leveraged for common application components, including
+the libraries listed previously.
+
+All components listed above are free to use for private and commercial use.
+
+\subsubsection{Products That Can Be Copied}
+
+There are no known products available that can be legally copied for use in this application.
+
 \subsection{New Problems}
 
+\subsubsection{Effects on the Current Environment}
+
+This application will change the way certain processes are preformed and these changes will impact
+the users.
+
+\textbf{Work Orders}
+
+The work order system will affect the way automotive mechanics document their work. The data will
+be inputted into the application therefore any failures can result in data loss.
+
+\textbf{Appointments}
+
+The appointments system will affect the way that both the customers and the receptionists schedule
+appointments. The application will track daily appointment schedules and report time conflicts. The
+application shall not lock the receptionist out of overriding the schedule.
+
+\textbf{Quotes}
+
+The quotes system will affect the way that both the customers and automotive repair shops
+communicate in the service quotation process. The quotes will now be communicated partially or
+completely through the application instead of completely in-person. Failure in this system may lead
+to a loss in data.
+
+\subsubsection{Effects on the Installed Systems}
+
+The application will be completely stand alone and will not be interfacing with any existing
+systems. The existing system may continue to coexist with the application at the user's discretion.
+
+\subsubsection{Potential User Problems}
+
+Any potential adverse reactions related to using the device in which application is being launched
+on (e.g., computer, mobile device, tablet, etc.) would extend to the use of this application. The
+application will not introduce any new adverse reactions to the user.
+
+\subsubsection{Limitations in the Anticipated Implementation Environment That May Inhibit the New Product}
+
+The database is not able to sustain the number of connections to serve all requests.
+
+\subsubsection{Follow-Up Problems}
+
+If any of the third-party integrations cease functioning, various components of the application
```

Comment:
Suggestion: Any failures or downtime of third-party integrations may impact the overall availability and operation of the application.